SingularityExecutorCleanup/src/main/java/com/hubspot/singularity/executor/cleanup/config/SingularityExecutorCleanupConfiguration.java

```diff
@@ -71,6 +72,12 @@
   @NotNull
   private String defaultS3Bucket = "";
 
+  @NotNull
+  private Optional<String> healthCheckResultFilePath = Optional.absent();
```

Comment:
why are these needed in executor cleanup? It should have no interaction with health checks